EIPS/eip-5851.md

```diff
@@ -0,0 +1,257 @@
+            },
+        "logic": ">=",
+        "value":"18"  
+    }); 
+    will correspond to the functionality that admin needs to adjust the standard claim for the identification SBT with tokenId = 1, based on the conditions described in the Claims array struct details.
+    **/
+
+    function changeStandardClaim(uint256 SBTID, Claim[] memory _claims) external returns (bool);
+
+    /// @notice function which uses the ZKProof protocol to validate the identity based on the given 
+    /// @dev it should only be called by the admin address.
+    /// @param SBTID is the Id of the SBT-based identity certificate for which admin wants to define the Claims.
+    /// @param claimer is the address that needs to be proven as the owner of the SBT defined by the tokenID.
+    /**
+    example: certify(0xA....., 10) means that admin assigns the DID badge with id 10 to the address defined by the `0xA....` wallet.
+    */
+    function certify(address claimer, uint256 SBTID) external returns (bool);
+
+    /// @notice function which uses the ZKProof protocol to validate the identity based on the given 
+    /// @dev it should only be called by the admin address.
+    /// @param SBTID is the Id of the SBT-based identity certificate for which the admin wants to define the Claims.
+    /// @param claimer is the address that needs to be proven as the owner of the SBT defined by the tokenID.
+    /* eg: revoke(0xfoo,1): means that KYC admin revokes the SBT certificate number 1 for the address '0xfoo'. */
+    function revoke(address certifying, uint256 SBTID) external returns (bool);
+
+```
+
+#### Events
+
+```solidity
+    /** 
+    * standardChanged
+    * @notice standardChanged MUST be triggered when claims are changed by the admin. 
+    * @dev standardChanged MUST also be triggered for the creation of a new SBTID.
+    e.g : emit StandardChanged(1, Claims(Metadata('age', 'uint', 'age of the person based on the birth date on the legal document' ), ">=", "18");
+    is emitted when the Claim condition is changed which allows the certificate holder to call the functions with the modifier, claims that the holder must be equal or more than 18 years old.
+    */
+    event StandardChanged(uint256 SBTID, Claim[] _claims);
+    
+    /** 
+    * certified
+    * @notice certified MUST be triggered when the SBT certificate is given to the certifying address. 
+    * eg: Certified(0xfoo,2); means that wallet holder address `0xfoo` is certified to hold a certificate issued with id 2, and thus can satisfy all the conditions defined by the required interface.
+    */
+    event Certified(address claimer, uint256 SBTID);
+    
+    /** 
+    * revoked
+    * @notice revoked MUST be triggered when the SBT certificate is revoked. 
+    * eg: Revoked( 0xfoo,1); means that entity user 0xfoo has been revoked to all the function access defined by the SBT ID 1.
+    */
+    event Revoked(address claimer, uint256 SBTID);
+}
+```
+
+## Rationale
+
```

Comment:
The `Rationale` section is not optional, though you can leave it as "TBD" for now:

```suggestion
TBD
```